Lib/test/test_dis.py

```diff
@@ -65,7 +65,7 @@ def cm(cls, x):
               8 STORE_ATTR               0 (x)
              10 LOAD_CONST               0 (None)
              12 RETURN_VALUE
-""" % (_C.cm.__code__.co_firstlineno + 2,)
+""" % (_C.cm.__code__.co_firstlineno + 1,)
```

Comment:
Why this is changed? I thought this PR shouldn't change `__code__.co_firstlineno`

Reply:
Thank you for pointing this out. I realized that my changes weren't actually updating the first line number in the compiler! I've fixed that.